src/gametest/java/ca/teamdman/sfm/gametest/SFMCorrectnessGameTests.java

```diff
@@ -1346,6 +1355,8 @@ public static void falling_anvil_xp_shard(GameTestHelper helper) {
             assertTrue(found.size() == 1, "should only be one item");
             assertTrue(found.get(0).getItem().is(SFMItems.EXPERIENCE_SHARD_ITEM.get()), "should be an xp shard");
             assertTrue(found.get(0).getItem().getCount() == 1, "should only be one");
+            found.forEach(Entity::discard);
```

Comment:
```suggestion
            //found.forEach(Entity::discard);
```
I want to see the items on the ground after the test

Reply:
we can do that for the last one but not the previous or itd interfere